apps/web/src/services/analytics/useMixpanel.ts

```diff
@@ -59,7 +59,11 @@ const useMixpanel = () => {
         console.info('[MixPanel] - User opted in')
       }
     } else {
-      mixpanel.opt_out_tracking()
+      try {
+        mixpanel.opt_out_tracking()
+      } catch (e) {
+        // do nothing, opt_out_tracking throws an error if tracking was never enabled
```

Comment:
I had to add this because otherwise the app doesn't work at all unless you add a Mixpanel env var (which most devs initially wouldn't do).